crates/sp-domains/src/lib.rs

```diff
@@ -391,13 +387,13 @@ impl<
         BlakeTwo256::hash_of(self)
     }
 
-    pub fn genesis(consensus_genesis_hash: Hash, genesis_state_root: DomainHash) -> Self {
+    pub fn genesis(genesis_state_root: DomainHash) -> Self {
         ExecutionReceipt {
             domain_block_number: Zero::zero(),
             domain_block_hash: Default::default(),
             domain_block_extrinsic_root: Default::default(),
             parent_domain_block_receipt_hash: Default::default(),
-            consensus_block_hash: consensus_genesis_hash,
+            consensus_block_hash: Default::default(),
```

Comment:
not sure why? Ideally, this should be the block hash in which the domain was instantiated. Is that not so ?

Reply:
Sorry, there should be a TODO here, but the change is due to the genesis domain being registered at the genesis block now, and we can't know the hash of the genesis block at that time.